### PR TITLE
Give users more info on lb4 discover.

### PR DESCRIPTION
**Pull request by SouthRibbleTech**
## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/Discovering-models.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ permalink: /doc/en/lb4/Discovering-models.html
 
 ## Synopsis
 
+Please note that the code below will not work without creating .js connector files. Also not that lb4 will only create .ts connector files. Your on your own, good luck.
+
```
Comment (raymondfeng) [inline docs/site/Discovering-models.md]: Can you elaborate the issue you ran into? Did you refer to an existing issue?

Comment (bajtos) [reply docs/site/Discovering-models.md]: +1 to what Raymond said.

The information in the note does not match my mental model of how model discovery works, it does not make sense to me.

Let's build a better understanding of the problem you @SouthRibbleTech are facing. Then we can provide a better advice than just "you are on your own" and perhaps even improve our tooling to support your scenario.

Comment (achrinza) [reply docs/site/Discovering-models.md]: I'm making an assumption, but I believe the culprit is this line:

image

While it does say the project must be built, it assumes that the readers know that they should run `npm run build`. Currently, there's no documentation on the default, generated npm scripts. The only documentation on them is on the [`@loopback/build` README ](https://github.com/strongloop/loopback-next/blob/master/packages/build/README.md).

We could potentially create a new doc page explaining the purpose of these scripts and how to use them, then link to it from the `Discovering Models` doc page.

**Comment by bajtos** [comment]
@SouthRibbleTech what's the status of this pull request? Did the explanation in https://github.com/strongloop/loopback-next/pull/3479#discussion_r347719403 clarified and resolved the problem for you? Are you still keen to improve our documentation to help the next person running into the same confusion?

**Comment by achrinza** [comment]
@SouthRibbleTech I believe #4670 provides additional clarity on the prerequisites of `lb4 discover`. If it doesn't, please feel free to open a new issue.

**Comment by achrinza** [comment]
Duplicate of #4670

**Pull request closed without merging**